templates/registration/login.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>Login</h1>
+    {% if error %}
+        <p style="color: red;">{{ error }}</p>
+    {% endif %}
+    <form action="{% url 'login' %}" method="post">
+        {% csrf_token %}
+        {{ form.as_p }}
+        <input type="submit" value="Submit">
+        <input type="hidden" name="next" value="{{ next }}">
```

Comment:
The `next` parameter is used to redirect users after a successful login. Ensure that this parameter is correctly set in the view or passed to the template context to avoid redirect issues.